glide-core/Cargo.toml

```diff
@@ -10,7 +10,7 @@ authors = ["Valkey GLIDE Maintainers"]
 [dependencies]
 bytes = "1"
 futures = "^0.3"
-redis = { path = "../submodules/redis-rs/redis", features = ["aio", "tokio-comp", "tokio-rustls-comp", "connection-manager","cluster", "cluster-async"] }
+redis = { path = "./redis-rs/redis", features = ["aio", "tokio-comp", "tokio-rustls-comp", "connection-manager","cluster", "cluster-async"] }
```

Comment:
Why do we need cluster? Cluster async is not enough?

Reply:
This is about adding the sources, not fixing them